docker/docker-compose.yml

```diff
@@ -1,34 +1,72 @@
 version: '3'
 services:
+  spark-submit:
+    image: metorikku/metorikku
+    container_name: spark-submit
+    ports:
+      - "4040:4040"
+    environment:
+      - SPARK_MASTER_HOST=spark-master
+      - SPARK_MASTER_PORT=7077
+      - SPARK_WEBUI_PORT=8080
+      - SPARK_UI_PORT=4040
+      - NUM_WORKERS=2
+      - JSON_LOG=true
+      - GRAPHITE_HOST=graphite.local
+      - GRAPHITE_PORT=3001
+      - SUBMIT_COMMAND=spark-submit --class com.yotpo.metorikku.Metorikku metorikku.jar -c examples/movies.yaml
+    entrypoint:
+      - /entrypoint-submit.sh
   spark-master:
     image: metorikku/metorikku
     container_name: spark-master
+    depends_on:
+      - spark-submit
     ports:
       - "8080:8080"
       - "7077:7077"
       - "6066:6066"
-      - "4040:4040"
     environment:
-    - SPARK_NODE=master
-    - SPARK_HOST=spark-master
-    - SPARK_PORT=7077
-    - SPARK_WEBUI_PORT=8080
-    - SPARK_UI_PORT=4040
-    - NUM_WORKERS=1
-    - JSON_LOG=true
-    - GRAPHITE_HOST=graphite.local
-    - GRAPHITE_PORT=3001
-    - SUBMIT_COMMAND=spark-submit --class com.yotpo.metorikku.Metorikku metorikku.jar -c examples/movies.yaml
+      - SPARK_MASTER_HOST=spark-master
+      - SPARK_MASTER_PORT=7077
+      - SPARK_WEBUI_PORT=8080
+      - JSON_LOG=true
+      - GRAPHITE_HOST=graphite.local
+      - GRAPHITE_PORT=3001
+    entrypoint:
+      - /entrypoint-master.sh
   spark-worker-1:
     image: metorikku/metorikku
     container_name: spark-worker-1
     depends_on:
-      - spark-master
+      - spark-submit
     ports:
       - "7078:7078"
     environment:
-      - SPARK_MASTER=spark://spark-master:7077
-      - SPARK_NODE=worker
-      - SPARK_PORT=7078
+      - SPARK_MASTER_HOST=spark-master
+      - SPARK_MASTER_PORT=7077
+      - SPARK_WORKER_PORT=7078
       - SPARK_WEBUI_PORT=8080
       - JSON_LOG=true
+      - GRAPHITE_HOST=graphite.local
+      - GRAPHITE_PORT=3001
```

Comment:
Can you please add all parameters here with a comment on them? (nomad, max retries etc.).
For now this is the only place we have documentation on the available params